mep/books/views.py

```diff
@@ -258,6 +260,55 @@ def get_breadcrumbs(self):
             ('Circulation', self.get_absolute_url())
         ]
 
+
+class WorkCardList(ListView, RdfViewMixin):
+    '''Card thumbnails for lending card associated with a single library
+    member.'''
+    model = Footnote
+    template_name = 'books/work_cardlist.html'
+    context_object_name = 'footnotes'
+
+    def get_queryset(self):
+        # find the associated book; 404 if not found
+        self.work = get_object_or_404(Work, slug=self.kwargs['slug'])
+
+        # find footnotes for events associated with this work
+        # that have images
+        return super().get_queryset() \
+                      .on_events() \
+                      .filter(Q(borrows__work__pk=self.work.pk) |
+                              Q(events__work__pk=self.work.pk) |
+                              Q(purchases__work__pk=self.work.pk)) \
+                      .filter(image__isnull=False) \
+                      .annotate(date=Coalesce('borrows__start_date',
+                                              'events__start_date',
+                                              'purchases__start_date')) \
+                      .prefetch_related('content_object', 'image') \
+                      .order_by(F('date').asc(nulls_last=True))
```

Comment:
wow, this is intense. nice work figuring it out!!

Reply:
I'm still annoyed that I wasn't able to consolidate footnotes so they are all linked to footnotes. I have a working migration for it! I just couldn't figure out a way to get the footnotes to show up in both admin edit forms. 😠 (Fought with it for a while and finally decided it wasn't a good use of my time.)